src/main/java/samza/examples/sql/samza-sql-groupby/src/main/sql/samza.sql

```diff
@@ -0,0 +1,5 @@
+insert into kafka.groupbyTopic
```

Comment:
Could we add a comment as to what we are doing here ?

-- Emit Page view counts collected grouped by page key in the last
-- 5 minutes at 5 minute interval and send the result to a kafka topic.
-- Using GetSqlField UDF to extract page key from the requestHeader.

Reply:
done